webapp/channels/src/components/admin_console/message_export_settings.tsx

```diff
@@ -31,18 +45,26 @@ export default class MessageExportSettings extends AdminSettings {
                 SMTPUsername: this.state.globalRelaySMTPUsername,
                 SMTPPassword: this.state.globalRelaySMTPPassword,
                 EmailAddress: this.state.globalRelayEmailAddress,
-
+                SMTPServerTimeout: this.state.globalRelaySMTPServerTimeout,
             };
         }
         return config;
     };
 
-    getStateFromConfig(config) {
-        const state = {
+    getStateFromConfig(config: AdminConfig) {
+        const state: State = {
             enableComplianceExport: config.MessageExportSettings.EnableExport,
             exportFormat: config.MessageExportSettings.ExportFormat,
             exportJobStartTime: config.MessageExportSettings.DailyRunTime,
-            canRunJob: config.MessageExportSettings.EnableExport,
```

Comment:
This seems to be a duplicate of `enableComplianceExport` above.

Reply:
nice find